### PR TITLE
Add new composite (true_color_with_night_fires) to GOES/ABI: True color (day) with fires (night)

### PR DESCRIPTION
**Pull request by simonrp84**
Last week I made a new composite for GOES/ABI that combines the `true_color` images for daytime with `cira_fire_temperature` at night.
Several people have since been in touch asking me how I made images using this composite, so I thought it worth adding to satpy main. This is only YAML changes and shouldn't affect any code / tests.

### CODE REVIEW

File: satpy/etc/enhancements/abi.yaml [hunk 1]
```diff
@@ -32,6 +32,18 @@ enhancements:
...
+
+  true_color_with_night_fires:
+    standard_name: true_color_with_night_fires
+    sensor: abi
+    operations:
+      - name: linear_stretch
+        method: !!python/name:satpy.enhancements.stretch
+      - name: sqrt
+        method: !!python/name:satpy.enhancements.gamma
+        kwargs: { gamma: 2.0 }
```
Comment (pnuu) [inline satpy/etc/enhancements/abi.yaml:44]: I would've expected the enhancement to be `crude` stretch from `0.0` to `1.0` as the individual composites are already stretched to their pre-defined values.

Comment (mraspaud) [reply satpy/etc/enhancements/abi.yaml:44]: I second that, what do you say @simonrp84 ?

Comment (simonrp84) [reply satpy/etc/enhancements/abi.yaml:44]: Sorry, overworked right now - will report back on this as soon as I can but I suspect you're right.

**Comment by gerritholl** [comment]
@simonrp84 Any updates on this?

**Comment by simonrp84** [comment]
Had forgotten about it, to be honest. I have no idea what the limits of the crude stretch should be! Will try with @pnuu's suggestion and see how it looks.

**Comment by simonrp84** [comment]
@pnuu and @mraspaud were correct, the enhancement should be crude. Have updated and this should be ready to merge now.

**Review by djhoese — APPROVED** [review]
LGTM

**Pull request merged**